CONTRIBUTING.md

```diff
@@ -10,12 +10,119 @@ We actively welcome your pull requests.
 5. Make sure your code lints.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
 
+## Forking, cloning, and running Rapid
+This section suggests a toolset and method to start hacking Rapid. However, you are welcome to use your own tools for the job. Here's what you could use:
+
+| Tool | Suggestion | Alternatives |
+|---|---|---|
+| 🟩 Node version manager | [nvm](https://github.com/nvm-sh/nvm) | [fish](https://github.com/jorgebucaran/nvm.fish), [n](https://github.com/tj/n), [volta](https://github.com/volta-cli/volta) |
+| 📝 Text editor | [VSCode](https://code.visualstudio.com/) | [Sublime](https://www.sublimetext.com/), [Vim](https://www.vim.org/), [Emacs](https://www.gnu.org/software/emacs/) |
+| 🧑‍💻 POSIX-compliant CLI | [Bash for Linux](https://www.gnu.org/software/bash/) | [WSL for Windows](https://learn.microsoft.com/en-us/windows/wsl/install), [Zsh the OSX default](https://www.zsh.org/) |
+| 🌐 Web browser | [Chrome](https://www.google.com/chrome/) | [Safari](https://www.apple.com/safari/), [Firefox](https://www.mozilla.org/en-US/firefox/new/), [Opera](https://www.opera.com/) |
+
+### Setting up Rapid
```

Comment:
I suggest changing the title to either 'How to Run the Project Locally' or 'Setting Up Rapid Locally' for a more descriptive and user-friendly heading.